pom.xml

```diff
@@ -0,0 +1,1315 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You need to change to gradle scripts. pom.xml has been removed from master branch.

Reply:
well, I found this last night and will change it to gradle  scripts  today.